### DISCUSSION TITLE
blog/hive-introducing-schema-policy

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1245] [Docs Discussions]
# blog/hive-introducing-schema-policy

New GraphQL-Hive feature for enfocring best-practices and schema-design styles.

https://the-guild.dev/blog/hive-introducing-schema-policy

**Comment by platzhersh** [comment]
Will this be enabled in the self-hosted GraphQL Hive aswell? Currently I do only see the empty tab, including a link to the documentation.
